Formula/prover9.rb

```diff
@@ -48,3 +56,41 @@ def install
     system bin/"mace4", "-f", testpath/"group2.in"
   end
 end
```

Comment:
```suggestion
end

```
Let's be consistent about formatting embedded patches.

Reply:
Fixed!